test/benchmarks/randomized_benchmarking.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+# pylint: disable=no-member,invalid-name,missing-docstring,no-name-in-module
+# pylint: disable=attribute-defined-outside-init,unsubscriptable-object
+
+"""Module for estimating randomized benchmarking."""
+
+import numpy as np
+import qiskit.ignis.verification.randomized_benchmarking as rb
+from qiskit.providers.basicaer import QasmSimulatorPy
+
+try:
+    from qiskit.compiler import transpile
+except ImportError:
+    from qiskit.transpiler import transpile
+
+
+def build_rb_circuit(nq, nseeds=1, length_vector=None,
+                     rb_pattern=None, length_multiplier=1,
+                     seed_offset=0, align_cliffs=False, seed=None):
+    """
+    Randomized Benchmarking sequences.
+    """
+    np.random.seed(seed)
+    rb_opts = {}
+    rb_opts['nseeds'] = nseeds
+    rb_opts['length_vector'] = length_vector
+    rb_opts['rb_pattern'] = rb_pattern
+    rb_opts['length_multiplier'] = length_multiplier
+    rb_opts['seed_offset'] = seed_offset
+    rb_opts['align_cliffs'] = align_cliffs
+
+    # Generate the sequences
+    try:
+        rb_circs, _ = rb.randomized_benchmarking_seq(**rb_opts)
+    except OSError:
+        skip_msg = ('Skipping tests for %s qubits because '
+                    'tables are missing' % str(nq))
+        print(skip_msg)
```

Comment:
In the case of an OSError we'll print this message to stdout and raise a NameError marking that benchmark as failed. Instead we should raise a `NotImplementedError` here to mark the benchmark as skipped (see: https://asv.readthedocs.io/en/latest/writing_benchmarks.html#setup-and-teardown-functions ):

```suggestion
        raise NotImplementedError(skip_msg)
```

Reply:
Done.